lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridge.cs

```diff
@@ -123,6 +127,7 @@ public class DriverBridge
             [Parameter(Value = typeof(DriverBridgeConfigurationOptions.DriverRestartActiveContextHandlers))] ISet<IObserver<IActiveContext>> driverRestartActiveContextHandlers,
             [Parameter(Value = typeof(DriverBridgeConfigurationOptions.DriverRestartRunningTaskHandlers))] ISet<IObserver<IRunningTask>> driverRestartRunningTaskHandlers,
             [Parameter(Value = typeof(DriverBridgeConfigurationOptions.TraceListenersSet))] ISet<TraceListener> traceListeners,
+            [Parameter(Value = typeof(EvaluatorConfigurationProviders ))] ISet<IConfigurationProvider> configurationProviders,
```

Comment:
You might need to add it to DriverBridgeConfiguration.ConfigurationModule so that t it can be set by the user.